src/main/java/org/junit/Assert.java

```diff
@@ -812,6 +812,41 @@ public static void assertNotSame(Object unexpected, Object actual) {
         assertNotSame(null, unexpected, actual);
     }
 
+    /**
+     * Asserts that two {@link CharSequence} instances have the same contents.
+     * If they do not, an {@link AssertionError} is thrown.
+     *
+     * @param expected the expected contents
+     * @param actual the object to compare to
+     */
+    public static void assertContentEquals(String expected, CharSequence actual)  {
+        assertContentEquals(null, expected, actual);
+    }
+
+    /**
+     * Asserts that two {@link CharSequence} instances have the same contents.
+     * If they do not, an {@link AssertionError} is thrown with the given message.
+     *
+     * @param message the identifying message for the {@link AssertionError}
+     * @param expected the expected contents
+     * @param actual the object to compare to
+     */
+    public static void assertContentEquals(String message, String expected, CharSequence actual) {
+        if (expected == null && actual == null) {
```

Comment:
As a potential user of this method, how should it handle null values?

(I assume short-circuiting the method if expected == actual is not controversial)